src/ResourceManagement/Scheduler/SchedulerManagement/packages.config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.Rest.ClientRuntime" version="1.0.2" targetFramework="net45" />
```

Comment:
I think you might looking at the old version.  I am currently using the following (this is what Storage uses).  I think I have to use this because of the AutoRest version I am using.  Is this not the case?
  
<package id="Microsoft.Rest.ClientRuntime" version="1.1.1" targetFramework="net45" />
  <package id="Microsoft.Rest.ClientRuntime.Azure" version="1.0.18-preview" targetFramework="net45" />